### ISSUE TITLE
Error : buf length isn't multiple of compact IP/PORTs (6 bytes)

### ISSUE THREAD

**Issue by munafio** [webtorrent/webtorrent#1922]
**What version of this package are you using?**
- "webtorrent": "^0.108.6"

**What operating system, Node.js, and npm version?**
- Node Version: v12.18.4
- NPM Version: v12.18.4
- OS: Linux Ubuntu 18.04 LTS

**What happened?**
I'm using WebTorrent to create a torrent downloader service with node.js .. I followed the docs step by step and it works but!! while downloading a torrent file (Sintel torrent file provided by the docs itself), the app crashes as the following : 
```
Downloading....
Downloading....
Downloading....

/home/username/Desktop/node-torrent/backend/node_modules/compact2string/index.js:22
    throw new Error("buf length isn't multiple of compact IP/PORTs (6 bytes)");
          ^
Error: buf length isn't multiple of compact IP/PORTs (6 bytes)
    at Function.compact2string.multi (/home/username/Desktop/node-torrent/backend/node_modules/compact2string/index.js:22:11)
    at utPex.onMessage (/home/username/Desktop/node-torrent/backend/node_modules/ut_pex/index.js:103:24)
    at Wire._onExtended (/home/username/Desktop/node-torrent/backend/node_modules/bittorrent-protocol/index.js:554:26)
    at Wire._onMessage (/home/username/Desktop/node-torrent/backend/node_modules/bittorrent-protocol/index.js:686:21)
    at Wire._write (/home/username/Desktop/node-torrent/backend/node_modules/bittorrent-protocol/index.js:590:12)
    at doWrite (/home/username/Desktop/node-torrent/backend/node_modules/readable-stream/lib/_stream_writable.js:409:139)
    at writeOrBuffer (/home/username/Desktop/node-torrent/backend/node_modules/readable-stream/lib/_stream_writable.js:398:5)
    at Wire.Writable.write (/home/username/Desktop/node-torrent/backend/node_modules/readable-stream/lib/_stream_writable.js:307:11)
    at Socket.ondata (_stream_readable.js:716:22)
    at Socket.emit (events.js:315:20)
[nodemon] app crashed - waiting for file changes before starting...
```
NOTE: I tried downloading another file, and it did not throw that error!

This issue related to `compact2string` package, because it throws an error instead of a warning message!
But why this is happening?? because of what? and how to be solved?

**What did you expect to happen?**
Not to throw an error

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by rogerg93** [comment]
Same issue here, is anyone working on a fix?

**Comment by hicom150** [comment]
There is already a PR https://github.com/webtorrent/ut_pex/pull/26 to fix it 😉
By the way, as this issue is duplicate of #1838 I'm going to close it so we can follow the progress there 👍

**Comment by rogerg93** [comment]
> There is already a PR [webtorrent/ut_pex#26](https://github.com/webtorrent/ut_pex/pull/26) to fix it 😉
> By the way, as this issue is duplicate of #1838 I'm going to close this so we can follow the progress there 👍

Thank you for fixing this issue. When can we expect it to be merged in a new version?

**Comment by munafio** [comment]
@hicom150 I know it's duplicated but no one did answer a good solution for that -_- and it's still there open with no solution!

**Comment by rogerg93** [comment]
@hicom150 I am still getting the same error, even though the pull request was merged to master. How should I update my project to use the latest changes you made?

**Comment by feross** [comment]
Released as 0.109.0
